docker/native-qemu.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
```

Comment:
This could be merged with some of the logic in #952in `base-runner.sh`.

Reply:
Actually, nevermind, they're pretty substantially different and this is the simplest logic available.